Node.js/GettingStarted/processcsv.py

```diff
@@ -0,0 +1,100 @@
+import csv
+import json
+from azure.storage.blob import BlockBlobService
+import datetime
+import os
+import argparse
+
+#### TODO: Get the configuration
+
+storage_acc_name = '<storage account name>'
```

Comment:
I have explained it in the node.js getting started article that this is not recommended way. This is a sample.